broker-cli/proto/broker.proto

```diff
@@ -538,6 +538,47 @@ message GetBlockOrdersRequest {
   string market = 1;
 }
 
+message TradeOrder {
+  string order_id = 1;
+  string block_order_id = 2;
+  string base_symbol = 3;
+  string counter_symbol = 4;
+  string side = 5;
+  string base_amount = 6;
+  string counter_amount = 7;
+  string state = 8;
+  string date_created = 9;
```

Comment:
These dates might be better as a submessage so that they can be managed separately.